src/rowbinary/tests.rs

```diff
@@ -100,6 +102,8 @@ fn sample_serialized() -> Vec<u8> {
         0x42, 0x54, 0x43, 0x00, /**/
         // [Array(Int32)] [-42, 42, -42, 42]
         0x04, 0xd6, 0x2a, 0xd6, 0x2a, /**/
+        // [Boolean] true
+        0x01,
```

Comment:
Please, add `/**/` before the newline to avoid joining by rustfmt when a new field is added

Reply:
Done